webservices/rest.py

```diff
@@ -215,6 +216,8 @@ def add_aggregate_resource(api, view, schedule, label):
 
 api.add_resource(download.DownloadView, '/download/<path:path>/')
 
+api.add_resource(legal.UniversalSearch, '/legal/search/')
```

Comment:
Hmm maybe could think of a better name. I mean universal in the context of legal documents, as in not AO-specific or reg-specific. Probably just `search` would be better for now.